tc/sgx/trusted_worker_manager/enclave/work_order_processor.cpp

```diff
@@ -126,10 +131,12 @@ namespace tcf {
             "encryptedRequestHash",
             "invalid request; failed to retrieve encrypted request hash");
 
+        // requesterSignature is optional field. Hence don't throw exception
+        // if param is not there or empty in the request.
         requester_signature = GetJsonStr(
             params_object,
```

Comment:
Could you please make changes to below optional fields
1. resultUri
2. notifyUri
3. workerEncryptionKey
4. dataEncryptionAlgorithm

Reply:
Done